### PR TITLE
8345335: Add excluded jdk_foreign tests to manual group

### PR DESCRIPTION
**Pull request by frkator**
@AlanBateman @mahendrachhipa @bwhuang-us @serhiysachkov @mcimadamore @JornVernee 

adding as manual tests

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8345335](https://bugs.openjdk.org/browse/JDK-8345335): Add excluded jdk_foreign tests to manual group (**Task** - P4)

### Reviewers
 * [Jorn Vernee](https://openjdk.org/census#jvernee) (@JornVernee - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/22580/head:pull/22580` \
`$ git checkout pull/22580`

Update a local copy of the PR: \
`$ git checkout pull/22580` \
`$ git pull https://git.openjdk.org/jdk.git pull/22580/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 22580`

View PR using the GUI difftool: \
`$ git pr show -t 22580`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/22580.diff">https://git.openjdk.org/jdk/pull/22580.diff</a>

</details>
<details><summary>Using Webrev</summary>

[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/22580#issuecomment-2520848252)
</details>

### CODE REVIEW

**Comment by shurymury** [comment]
LGTM

File: test/jdk/TEST.groups [hunk 1]
```diff
@@ -620,7 +620,9 @@ jdk_core_manual_no_input = \
     java/util/Vector/Bug8148174.java \
     com/sun/net/httpserver/simpleserver/CommandLinePortNotSpecifiedTest.java \
     com/sun/net/httpserver/simpleserver/jwebserver/CommandLinePortNotSpecifiedTest.java \
-    javax/xml/jaxp/datatype/8033980/GregorianCalAndDurSerDataUtil.java
+    javax/xml/jaxp/datatype/8033980/GregorianCalAndDurSerDataUtil.java \
+    java/foreign/TestMatrix.java \
+    java/foreign/TestUpcallStress.java
```
Comment (msheppar) [inline test/jdk/TEST.groups:625]: I note that the test TestUpcallStress.java doesn't have a manual tag in the run directive ?

Comment (AlanBateman) [reply test/jdk/TEST.groups:625]: Might be a mistake, TestMatrix is the manual test.

Comment (JornVernee) [reply test/jdk/TEST.groups:625]: TestUpcallStress.java was supposed to run in tier4, which is a catch-all for any test not run in lower tiers:

```
# Everything not in other tiers
tier4 = \
    / \
   -:tier1 \
   -:tier2 \
   -:tier3
```

But, it doesn't seem like TestUpcallStress is actually being run in tier4...

File: test/jdk/TEST.groups [hunk 2]
```diff
@@ -92,7 +92,8 @@ tier3 = \
     -:jdk_svc_sanity \
     -:svc_tools \
     :jdk_jpackage \
-    :jdk_since_checks
+    :jdk_since_checks \
+    :jdk_foreign_stress
```
Comment (JornVernee) [inline test/jdk/TEST.groups:96]: Shouldn't it be in tier4?

File: test/jdk/TEST.groups [hunk 3]
```diff
@@ -381,9 +382,11 @@ jdk_svc = \
 
 jdk_foreign = \
     java/foreign \
-    jdk/internal/reflect/CallerSensitive/CheckCSMs.java \
-    -java/foreign/TestMatrix.java \
-    -java/foreign/TestUpcallStress.java
+    jdk/internal/reflect/CallerSensitive/CheckCSMs.java
```
Comment (JornVernee) [inline test/jdk/TEST.groups:385]: Please exclude the `jdk_foreign_stress` group from the regular `jdk_foreign` test group as well. We regularly run `jdk_foreign` as part of the normal dev cycle.

**Review by JornVernee — APPROVED** [review]
Changes for jdk_foreign tests look good.

**Comment by msheppar** [comment]
The JBS ticket title should be changed to reflect the changes made

**Pull request closed without merging**